### DISCUSSION TITLE
Exploring the New 6502 Compiler Based on LLVM — 8bitworkshop documentation

### DISCUSSION THREAD

**Discussion by giscus** [sehugg/8bitworkshop#176] [General]
# Exploring the New 6502 Compiler Based on LLVM — 8bitworkshop documentation

Today, homebrew developers enjoy the convenience of programming the 6502 in C. However, the 6502 is a very different architecture than modern CPUs, and was never designed for C programming. The 650...

https://8bitworkshop.com/docs/posts/2023/llvm-mos-sdk-6502-c-compiler.html

**Comment by eharvey71** [comment]
Hey there! I'm fascinated and blown away by 8bitworkshop. I can't fathom how this is all working in a browser, so I'm exploring WebAssembly and related technologies. Anyway, have you considered adding the TI99/4A? One excellent project (JS99'er) is a polished example of an implementation using TypeScript: https://js99er.net/#/

This is fun stuff! Thank you.
